docs/reference/tab-widgets/troubleshooting/disk/increase-data-node-capacity.asciidoc

```diff
@@ -0,0 +1,110 @@
+name of your deployment.
++
+. If autoscaling is available but not enabled, please enable it. You can do this by clicking the button
+`Enable autoscaling` on a banner like the one below:
++
+[role="screenshot"]
+image::images/troubleshooting/disk/autoscaling_banner.png[Autoscaling banner,align="center"]
++
+Or you can go to `Actions > Edit deployment`, check the checkbox `Autoscale` and click `save` at the bottom of the page.
++
+[role="screenshot"]
+image::images/troubleshooting/disk/enable_autoscaling.png[Enabling autoscaling,align="center"]
+
+. If autoscaling has succeeded the cluster should return to `healthy` status. If the cluster is still out of disk,
+please check if autoscaling has reached its limits. You will be notified about this by the following banner:
++
+[role="screenshot"]
+image::images/troubleshooting/disk/autoscaling_limits_banner.png[Autoscaling banner,align="center"]
++
+or you can go to `Actions > Edit deployment` and look for the label `LIMIT REACHED` as shown below:
++
+[role="screenshot"]
+image::images/troubleshooting/disk/reached_autoscaling_limits.png[Autoscaling limits reached,align="center"]
++
+If you are seeing the banner click `Update autoscaling settings` to go to the `Edit` page. Otherwise, you are already
+in the `Edit` page, click `Edit settings` to increase the autoscaling limits. After you perform the change click `save`
+at the bottom of the page.
+
+// end::cloud[]
+
+// tag::self-managed[]
+In order to increase the data node capacity in your cluster, you will need to calculate the amount of extra disk space
+needed.
+
+. First, retrieve the relevant disk thresholds that will indicate how much space should be available. The
+relevant thresholds are the <<cluster-routing-watermark-high, high watermark>> for all the tiers apart from the frozen
+one and the <<cluster-routing-flood-stage-frozen, frozen flood stage watermark>> for the frozen tier. The following
+example demonstrates disk shortage in the hot tier, so we will only retrieve the high watermark:
++
+[source,console]
+----
+GET _cluster/settings?include_defaults&filter_path=*.cluster.routing.allocation.disk.watermark.high*
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "defaults": {
+    "cluster": {
+      "routing": {
+        "allocation": {
+          "disk": {
+            "watermark": {
+              "high": "90%",
+              "high.max_headroom": "150GB"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+----
+// TEST[skip:illustration purposes only]
++
+The above means that in order to resolve the disk shortage we need to either drop our disk usage below the 90% or have
+more than 150GB available, read more on how this threshold works <<cluster-routing-watermark-high, here>>.
+
+. The next step is to find out the current disk usage, this will indicate how much extra space is needed. For simplicity,
```

Comment:
I think a semi-colon is more appropriate after 'current disk usage', since the rest of the sentence is basically a complete sentence on its own.

Reply:
done